pkg/aspectsupport/logger/manager.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logger
+
+import (
+	"fmt"
+
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/golang/protobuf/ptypes/struct"
+	"google.golang.org/genproto/googleapis/rpc/code"
+	"istio.io/mixer/pkg/aspect"
+	"istio.io/mixer/pkg/aspect/logger"
+	"istio.io/mixer/pkg/aspectsupport"
+	"istio.io/mixer/pkg/attribute"
+	"istio.io/mixer/pkg/expr"
+)
+
+type (
+	manager struct{}
+
+	executor struct {
+		inputs map[string]string
+		aspect logger.Aspect
+	}
+)
+
+// NewManager returns an aspect manager for the logger aspect.
+func NewManager() aspectsupport.Manager {
+	return &manager{}
+}
+
+func (m *manager) NewAspect(c *aspectsupport.CombinedConfig, a aspect.Adapter, env aspect.Env) (aspectsupport.AspectWrapper, error) {
+	// cast to logger.Adapter from aspect.Adapter
+	logAdapter, ok := a.(logger.Adapter)
+	if !ok {
+		return nil, fmt.Errorf("Adapter of incorrect type. Expected logger.Adapter got %#v %T", a, a)
```

Comment:
good catch. i thought lowercase was preferred, but thought I saw that we weren't doing that elsewhere and then thought I my memory was bad. I'll update here and below.